pcap/pcap.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2014 Damjan Cvetko. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the same directory
+// containing this source file.
+
+package pcap
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/brimsec/zq/pkg/nano"
+	"github.com/brimsec/zq/pkg/peeker"
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+)
+
+// Reader wraps an underlying io.Reader to read packet data in PCAP
+// format.  See http://wiki.wireshark.org/Development/LibpcapFileFormat
+// for information on the file format.
+//
+// We currenty read v2.4 file format with nanosecond and microsecdond
```

Comment:
```suggestion
// We currenty read v2.4 file format with nanosecond and microsecond
```

Reply:
Also I thought Libpcap classic (non ng) doesn't support nanosecond time resolution.

Reply:
Ok, that typo is in the copied-in code.  Will fix.